mmv1/third_party/terraform/services/storage/resource_storage_bucket_test.go.erb

```diff
@@ -323,6 +323,12 @@ func TestAccStorageBucket_lifecycleRulesMultiple(t *testing.T) {
 			},
 			{
 				Config: testAccStorageBucket_lifecycleRulesMultiple_update(bucketName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
I think the `ImportStateVerify` covers this for you. Do you know if these checks fail without your fix in place?

Reply:
Well, I thought of not explicitly checking fields but currently we are having complex issues with lifecycle rules. From that issues I found that there are many problems underneath in terraform core that we can't control and sometimes it creates unexpected behaviors. That's why I put this checks explicitly for lifecycle rules. If we don't any problems with tests or we don't require any separate check for TypeSet then I will remove it!
If It is okay to remove them I can remove them from the PR.

Reply:
Hmm yea, the TypeSet does make it a bit weird, and I'm actually wondering if the indexes in these check would be useful in the way you are hoping. I think they could be removed, but like I said, it would be nice to have something in place that fails with the previous (broken) implementation.

Reply:
Yes, I modified one of the update config to change value of  `custom_time_before` field and added `noncurrent_time_before` fields in the tests configs. That should be enough to tests that field are being modified properly. The indexes were working, I tried changing field values and tests were failing correctly as expected.